client/my-sites/plans/doughnut-chart/index.jsx

```diff
@@ -0,0 +1,17 @@
+import { Component } from 'react';
+
+import './style.scss';
+
+class DoughnutChart extends Component {
+	render() {
+		const { progress, text } = this.props;
+
+		return (
+			<div className="doughnut-chart__wrapper" style={ { '--percentage': progress * 100 } }>
+				{ text }
+			</div>
+		);
+	}
+}
+
+export default DoughnutChart;
```

Comment:
Another functional component + TypeScript candidate.
```suggestion
export default const DoughnutChart = ( props ) => {
	const { progress, text } = props;

	return (
		<div className="doughnut-chart__wrapper" style={ { '--percentage': progress * 100 } }>
			{ text }
		</div>
	);
};
```